torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1734,6 +1734,32 @@ int listList(Stack& stack) {
   return 0;
 }
 
+template <typename T>
+int listContains(Stack& stack) {
+  auto key = pop(stack).to<T>();
+  auto list = pop(stack).to<c10::List<T>>();
+  for (const T& item : list) {
+    if (item == key) {
+      push(stack, true);
+      return 0;
+    }
+  }
+  push(stack, false);
+  return 0;
+}
+
+Operation listContainsStr(const Node* node) {
+  auto list = node->inputs().at(0)->type()->expect<ListType>();
+
+  if (!list->getElementType()->isSubtypeOf(StringType::get())) {
```

Comment:
I don't think this is needed, as the schema matching already check the element type is a string type or not. If we want this better error message, we should apply this to all primitive types rather than only string type.